### DISCUSSION TITLE
Web crawl reveals unknown pages...

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#6142] [General]
Hi

I did a web crawl (used Screaming Frog) and noticed 78 pages where surfaced that shouldn't exist.

I think what's happening is a custom route I created is somehow generating additional pages, that match (but shouldn't).

Here's the custom route
```
Route::statamic('/divisions/{segment_2}/contact', 'divisions/contact');
```

Here's an example of one of the pages that appears to be using the custom route. The page is using the `'divisions/contact'` view
```
website.com/divisions/focusing-on-the-future/contact
```

`focusing-on-the-future` is actually a `news` entry, not sure why it's being used in `divisions`...

How would I go about fixing this? There's no errors in laravel.log and the page loads. It's just using a `url` that shouldn't be there?

`divisions:collection` has this route `/divisions/{slug}` and `news:collection` has this route `/news/{slug}`. Perhaps this is the issue?

I thought the route in the custom route `/divisions/{segment_2}/contact` would be enough to isolate divisions from news, but perhaps not?

**Comment by jasonvarga** [comment]
If you're using Screaming Frog, can't you see where a certain link was discovered? The "Inlinks" feature, I think.

(I haven't used the software, I'm just looking at their website and video)

**Reply by stephenmeehanuk** [reply]
Cheers. Figured it out, an `if` was being triggered in the wrong part of the site. That was responsible for the unwanted path. Fixed!
